tests/functional/spec/features/calculated_summary.spec.js

```diff
@@ -192,8 +190,23 @@ describe("Feature: Calculated Summary", () => {
       expect($(NumberTotalPlaybackPage.sixthNumberAnswer()).getText()).to.contain("45.67");
     });
 
-    it("Given I confirm the total and am on the summary, When I edit and change an answer, Then I must re-confirm the calculated summary page which is dependent on the change before I can return to the summary", () => {
+    it("Given I complete every calculated summary, When I go to the next page, Then I should the see the piped calculated summary total for each summary", () => {
```

Comment:
Maybe not use next page as that could change in time, it doesn't necessarily have to be straight after the calculated summary